keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1724,6 +1752,22 @@ Pod scheduling and startup are more important. However, expected performance
 will depend on how resources are used (for example, how often new Pods are
 created), therefore it is impossible to predict what reasonable SLOs might be.
 
+The resource manager component will do its work similarly to the 
+existing volume manager, but the overhead and complexity should
+be lower:
+
+* Resource preparation should be fairly quick as in most cases it simply
+  creates CDI file 1-3 Kb in size. Unpreparing resource usually means
+  deleting CDI file, so it should be quick as well.
```

Comment:
Beware that this ignores that "real" resource drivers may have to do more work to update resource tracking. We don't need to highlight this here, but it may come up in the KEP review.

Reply:
I understand that. However, we stated in the KEP that recommended way is to do all heavy work on allocation stage. We can point reviewers to that, I believe.